src/libs/actions/Report.js

```diff
@@ -414,7 +414,7 @@ function addActions(reportID, text = '', file) {
     // Update the report in Onyx to have the new sequence number
     const optimisticReport = {
         maxSequenceNumber: newSequenceNumber,
-        lastMessageTimestamp: Date.now(),
+        lastMessageTimestamp: lastAction.timestamp,
```

Comment:
Should this be `Date.parse(lastAction.created)`?

Reply:
First there is no, `lastAction.timestamp`. i think its `lastAction.created`. Then yes `Date.parse(lastAction.created)` looks good to me since i believe it should retain the milliseconds precision. Pls test and confirm.